project_timebox/models/project_task.py

```diff
@@ -0,0 +1,15 @@
+
+from odoo import models, fields
+
+
+class Task(models.Model):
+    _inherit = "project.task"
+
+    timebox = fields.Selection([
```

Comment:
Think we need to create an external model and a many2one here ?

Reply:
Done